Arrowgene.Ddon.Server/GameLogicSetting.cs

```diff
@@ -17,16 +17,25 @@ public class GameLogicSetting
         [DataMember(Order = 1)]
         public double AdditionalCostPerformanceFactor { get; set; }
 
+        /// <summary>
+        /// Controls whether to use the possibly expensive lobby handling.
```

Comment:
I am not sure I follow why this is more expensive than what we currently do. Shouldn't this be less expensive since we only send to the players in the actual hub?

Reply:
Before, context packets were sent only on login, now packets are moved around in both directions on entering/exiting hub areas